### ISSUE TITLE
Add 'Mastodon' and 'Blue Sky' social options for Events and People

### ISSUE THREAD

**Issue by VickyGarnett** [DARIAH-ERIC/dariah-campus#1282]
### Bug report

Hi @ttasovac @stefanprobst - while giving a demo of how to put an event together today, I noticed the 'Social' section for the metadata for any event is showing 'Twitter', but doesn't have any others.  I wondered, given the way Twitter (or if you prefer, 'X', blurgh!) is going, and how many of our community are moving towards Mastodon or Blue Sky, should we add those as an option on the 'Event' creation, and also for individual 'People' in their profiles?  Do we also want to add LinkedIn?

**Comment by stefanprobst** [comment]
yes, see https://dariah.slack.com/archives/C07D92SREUE/p1721651379478899 and https://dariah.slack.com/archives/C07D92SREUE/p1721651743797769
